orangecontrib/imageanalytics/image_embedder.py

```diff
@@ -88,7 +88,7 @@ class ImageEmbedder(Http2Client):
     CANNOT_LOAD = "cannot load"
 
     def __init__(self, model="inception-v3", layer="penultimate",
-                 server_url='api.biolab.si:8080'):
+                 server_url='api.garaza.io:443'):
```

Comment:
Maybe it's better to switch to a different URL with a separate PR, and not add it here?
Note that this URL is also temporary and will probably change as soon as we promote the new test k8s cluster into a production.